Release Notes: Add missing report bug fixes (#53597)
grafana/grafana: docs/sources/release-notes/release-notes-9-0-7.md

### Bug fixes

- **GrafanaUI:** Render PageToolbar's leftItems regardless of title's presence. [#53285](https://github.com/grafana/grafana/pull/53285), [@Elfo404](https://github.com/Elfo404)
- **Reports:** Fix inconsistency reports. (Enterprise)

> - **Reports:** Set uid when we don't receive it in the query. (Enterprise)

Reviewer: ```suggestion
- **Reports:** Set UID when it's not received in the query. (Enterprise)
```
